crates/sui-framework/packages/sui-framework/sources/funds_accumulator.move

```diff
@@ -71,7 +71,7 @@ public fun withdrawal_join<T: store>(withdrawal: &mut Withdrawal<T>, other: With
 // TODO When this becomes `public` we need
 // - custom verifier rules for `T`
 // - private generic rules for `T`
-public(package) fun redeem</* internal */ T: store>(withdrawal: Withdrawal<T>): T {
+public fun redeem</* internal */ T: store>(withdrawal: Withdrawal<T>): T {
```

Comment:
This will always be internal and needs to remain public package unless you add it to the private generics verifier (which we don't need right now because of it is only Balance)